.dockerignore

```
@@ -0,0 +1,7 @@
+.devcontainer
+.github
+.mypy_cache
+deploy-scripts
+examples
+setup-ai-verify
+third-party
```

Can we have files end with a (single) EOL?

```suggestion
third-party

```

---

solved